encoding/gocode/testdata/generics.txtar

```diff
@@ -0,0 +1,112 @@
+-- in.cue --
+package generate
+
+import "github.com/grafana/thema"
+
+thema.#Lineage
+name: "defaultchange"
+seqs: [
+        {
+            schemas: [
+                {
+                    #Generic: string | bool
```

Comment:
I think we should avoid using the word "Generic" here (and in general) in order to avoid anyb possible ambiguity with language generics